src/task/methods/jn_ap_op2/config.vsh.yaml

```diff
@@ -0,0 +1,31 @@
+__merge__: ../../api/comp_method.yaml
+
+functionality:
+  name: jn_ap_op2
+  info:
+    label: A deep NN model proposed by Antoine Passiemier and Jalil Nourisa
+    summary: "Deep learning architecture composed of 2 modules: a sample-centric MLP and a gene-centric MLP"
+    description: |
+      We first encode each sample using leave-one-out encoder based on compound and cell type. This produces X with the dimension of n_samples, n_genes, n_encode,
+      where n_encode is 2. Then, X is passed to a MLP1 sample-wise with input of n_samples, n_genes*n_encode, which outputs the same dimension data.
+      The purpose of this MLP is to learn inter-gene relationships. Then, we group the output of MLP1 with X (original encoded data) and feed it
+      to MLP2 which receives n_smaples*n_genes, (n_encode + n_encode) and results n_samples*n_genes. This MLP2 trains on each (compound, cell_type, gene)
+      combination. This is to overcome the underdetermination problem due to lack of sufficient (compound, cell_type) samples. 
+    documentation_url: https://www.kaggle.com/competitions/open-problems-single-cell-perturbations/discussion/461159
+    repository_url: https://github.com/AntoinePassemiers/Open-Challenges-Single-Cell-Perturbations
+
```

Comment:
Right now the unit test on the CI is failing because the execution time is larger than 30 minutes. Could you add an argument to this config which has a default value (for regular execution) and test default value (just for passing the unit test)?

```yaml
  arguments:
    - type: integer
      name: --epochs
      default: 100
      info:
        test_default: 10
```